### DISCUSSION TITLE
Brackets is extremely slow on Mac - is there something I can do

### DISCUSSION THREAD

**Discussion by altcointrading** [brackets-cont/brackets#255] [Q&A]
I have been looking for an alternative to Atom, sadly, and Brackets is OK for what I need, except it is so extremely slow. I have been using it about 2 weeks, it freezes at least 3 times a day, will not open anything from Filezilla, switching between folders in a project takes enough time to make a coffee. 

Is there any config to do? I found some discussion on config for Windows where people had this issue.

I can't have the wrong chip can I -- 2.3 GHz Dual-Core Intel Core i5 -- sometimes apps act crazy like this on my M1 chip machine, but never on this one.

**Comment by charlypa** [comment]
@altcointrading 
We have modernized brackets. We released it yesterday. The new editor is Phoenix Code, Now you can download native m1 and intel dmg instead of relying on old Brackets x86 binaries
visit https://phcode.io/
